portal-impl/src/com/liferay/portal/service/impl/OrganizationLocalServiceImpl.java

```diff
@@ -318,6 +321,14 @@ public Organization addOrganization(
 		organization.setCountryId(countryId);
 		organization.setStatusListTypeId(statusListTypeId);
 		organization.setComments(comments);
+
```

Comment:
please remember to update also the _validate method to avoid the check on the country mandatory or not if we are creating an incomplete model. (so add the new condition to the already existing if statement)